tests/FakeItEasy.Specs/OrderedCallMatchingSpecs.cs

```diff
@@ -85,12 +85,12 @@ public static void OrderedAssertionsOutOfOrder(IFoo fake, Exception exception)
                 .x(() => exception.Should().BeAnExceptionOfType<ExpectationException>().WithMessage(@"
 
   Assertion failed for the following calls:
-    'FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 1)' repeated exactly twice
-    'FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 2)' repeated at least once
-    'FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 3)' repeated at least once
+    'FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 1)' exactly twice
+    'FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 2)' at least once
+    'FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 3)' at least once
   The calls were found but not in the correct order among the calls:
     1: FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 3)
-    2: FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 1) repeated 2 times
+    2: FakeItEasy.Specs.OrderedCallMatchingSpecs+IFoo.Bar(baz: 1) (2 times)

```

Comment:
I don't think the difference between how the expected calls are rendered (with single quotes) and the found ones (without) is desirable, but I figure it's a different issue, so I wasn't going to attack at this point.

Reply:
Agreed. Shouldn't we include parenthesis for the expected call count, though?

Reply:
```

Reply:
For some reason the `()`s around the "at least once" don't sit well with m.e. I don't know why.

Reply:
Dropping the parens everywhere sounds good to me 👍